x-pack/platform/packages/shared/kbn-entities-schema/src/schema/v2/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface EntityV2 {
```

Comment:
Who is the onwer of this one?

Reply:
[obs-entities](https://github.com/elastic/kibana/blob/95af8758005f02a3ea56594ba6df641c02d084c0/.github/CODEOWNERS#L842). V2 is not used by transforms